plugins/catalog-backend-module-ldap/tsconfig.cli.json

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": "@backstage/cli/config/tsconfig.json",
```

Comment:
No extra tsconfig files please, they will confuse people

Reply:
Where is the issue where people were confused? If there is no issue, who does it confuse? 🤔 Or maybe a more valuable question: what alternative to do you propose to compile my TypeScript-based CLI script into JS that is external to the single-file `index.cjs.js` output when this plugin is built?

Reply:
There is no issue where people were confused because we haven't allowed additional tsconfig files :grin:
The trouble is the expectation that changing things in the file has an effect on type checking for example.

Either way I think the thing to figure out is the parallel build rather than the existence of this file, let's bounce some ideas around that